qcodes/instrument/parameter.py

```diff
@@ -855,7 +855,8 @@ def __init__(self, name: str,
 
         # Enable set/get methods if get_cmd/set_cmd is given
         # Called first so super().__init__ can wrap get/set methods
```

Comment:
Can you update the doc string to explain how this really works?

Reply:
sure thing